vendors/microchip/secure_elements/app/example_trust_chain_tool/README.md

```diff
@@ -25,47 +25,75 @@ PKI system) and an intermediate certificate authority that mirrors the signing
 hardware security modules (HSM) that are used in the Microchip facility during
 manufacturing of security devices.
 
-### Create the Root CA
+In this flow, you will create a Root CA, a Signing CA, and finally a device certificate
```

Comment:
I think we should rewrite the previous paragraph, above: "The first step is to setup a certificate chain of trust that resembles a production IoT ecosystem. For lab testing purposes, we'll create a self-signed root certificate authority and an intermediate certificate authority."

Reply:
done